packages/preview/oxdraw/0.1.0/typst.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "oxdraw"
```

Comment:
You seem to be involved with the `oxdraw` project. Usually for organization templates we require email verification for official packages, but here it seems easier to keep this on github. If this should be an official package could @RohanAdwankar leave a comment if that's ok? Otherwise the name would have to change, either by adding some non-descriptive prefix or by getting otherwise creative. See [our naming rules](https://github.com/typst/packages/blob/main/docs/manifest.md#naming-rules)

Reply:
Great!